forward/forwarder_test.go

```diff
@@ -145,8 +255,7 @@ func (s *ForwarderTestSuite) TestKeysDiverged() {
 func (s *ForwarderTestSuite) TestRequestTimesOut() {
 	var ping Ping
 
-	s.sender.lookup = "127.0.0.2:3001"
-	dest, err := s.sender.Lookup("some key")
+	dest, err := s.sender.Lookup("unreachable")
```

Comment:
Just FYI, on Ubuntu — and probably other distros — the entire 127.0.0.0/8 block is an alias for the loopback interface.

E.g.:

```
# nc -l 0.0.0.0 3002 &
[1] 22932
# telnet 127.184.192.2 3002
Trying 127.184.192.2...
Connected to 127.184.192.2.
Escape character is '^]'.
^]
telnet> quit
Connection closed.

[1]+  Stopped                 nc -l 0.0.0.0 3002
# 
```

So if something is listening on `0.0.0.0:3002` then it will respond when you connect to `127.0.0.2:3002`.